### PR TITLE
Next master

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
@CohenCyril do you know why the nix job picks up an old version of coq-elpi and not this branch?

File: .nix/config.nix [hunk 1]
```diff
@@ -10,8 +10,6 @@
       hierarchy-builder.override.version = "coq-elpi-2";
       hierarchy-builder-shim.job = false;
 
-      coq-elpi.override.version = "master";
```
Comment (CohenCyril) [inline .nix/config.nix:13]: @gares this was asking the action to test the master branch explicitly.

Comment (gares) [reply .nix/config.nix:13]: Thanks

**Comment by gares** [comment]
Thanks @CohenCyril I now see it passes MC and OOthm, so I'm OK releasing this branch.
But there is a problem with finmap, otherwise I'd love to also test MCA before releasing.

**Pull request merged**